runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/widgets/FieldTemplate/FieldTemplate.tsx

```diff
@@ -41,14 +43,14 @@ const FieldTemplate = ({
             {children}
             {displayLabel && rawDescription ? (
                 <Typography variant="caption" color="textSecondary">
-                    {rawDescription}
+                    {moment.locale() === "en" ? rawDescription : translate(`Process.BuildView.Modeler.Widgets.FieldTemplate.AssignTheOutputToVariable`)}
```

Comment:
are these `rawDescription` and `error` constants? because if it's not (I think it is) we cannot add constant polish translation

Reply:
You're right. I forgot about it. That was the reason I created checkIfKeyExists. You'll see changes in new PR.